OpenEdXMobile/res/layout/row_course_outline_list.xml

```diff
@@ -93,8 +93,6 @@
                     android:id="@+id/row_subtitle_due_date"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:ellipsize="end"
-                    android:singleLine="true"
```

Comment:
Due dates' text isn't wrapping/expanding properly when it goes multi-line.

![text_wrapping](https://user-images.githubusercontent.com/1710804/30053136-31ca45fc-9241-11e7-9c3f-8bfd2076101f.png)